independent-publisher-connectors/EONET by NASA/apiDefinition.swagger.json

```diff
@@ -0,0 +1,977 @@
+                        "items": {
+                          "type": "object",
+                          "properties": {
+                            "id": {
+                              "type": "string",
+                              "description": "The source ID.",
+                              "title": "Source ID"
+                            },
+                            "url": {
+                              "type": "string",
+                              "description": "The source URL.",
+                              "title": "Source URL"
+                            }
+                          }
+                        },
+                        "description": "One or more categories assigned to the event.",
+                        "title": "Sources"
+                      },
+                      "geometry": {
+                        "type": "array",
+                        "items": {
+                          "type": "object",
+                          "properties": {
+                            "magnitudeValue": {
+                              "type": "number",
+                              "format": "float",
+                              "description": "The magnitude value.",
+                              "title": "Magnitude Value"
+                            },
+                            "magnitudeUnit": {
+                              "type": "string",
+                              "description": "The magnitude unit.",
+                              "title": "Magnitude Unit"
+                            },
+                            "date": {
+                              "type": "string",
+                              "description": "The date.",
+                              "title": "date"
+                            },
+                            "type": {
+                              "type": "string",
+                              "description": "The geometry type.",
+                              "title": "type"
+                            },
+                            "coordinates": {
+                              "type": "array",
+                              "items": {
+                                "type": "number",
+                                "format": "float",
+                                "description": "The geometry coordinates.",
+                                "title": "Coordinate value"
+                              },
+                              "description": "The geometry coordinates.",
+                              "title": "Coordinates"
+                            }
+                          }
+                        },
+                        "description": "One or more event geometries are the pairing of a specific date/time with a location. The date/time will most likely be 00:00Z unless the source provided a particular time. The geometry will be a GeoJSON object of either type \u201cPoint\u201d or \u201cPolygon.\u201d",

```

Comment:
@troystaylor Still, there are unicodes in the swagger. Can you search with "\u", you will find all of them.

Reply:
Sorry.